app/views/keywords/_url_table.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if not urls.empty? %>
+  <table class="table">
+    <thead>
+    <tr>
+      <th scope="col">#</th>
+      <th scope="col">URL</th>
+    </tr>
+    </thead>
+    <tbody>
+    <% urls.each_with_index do |url, index| %>
```

Comment:
Indentation
```suggestion
      <% urls.each_with_index do |url, index| %>
```

Reply:
Fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)